mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -434,8 +434,39 @@ private[python] class PythonMLLibAPI extends Serializable {
     new Normalizer(p).transform(rdd)
   }
 
+  private[python] class StandardScalerModelWrapper(model: StandardScalerModel)
```

Comment:
@Lewuathe The wrapper doesn't do anything different that `StandardScalerModel`. Do we really need it?